composer.json

```diff
@@ -21,7 +21,8 @@
 
 	"require-dev": {
 		"satooshi/php-coveralls": "dev-master",
-		"phpunit/php-file-iterator": "1.3.3"
+		"phpunit/php-file-iterator": "1.3.3",
+		"phpunit/phpunit": "~4.0"
```

Comment:
Then I guess we keep it, since people will keep trying to add it over and over again if we remove it now :wink: